### PR TITLE
style: format golang import package code style

### PR DESCRIPTION
**Pull request by czybjtu**
## Summary

Apply `gofmt` changes to the code base.

### CODE REVIEW

**Comment by yqlbu** [comment]
@czybjtu not necessary to do so.

**Comment by czybjtu** [comment]
IMO, it will improve the readability of the code.  And most code editors now automatically format the code when saving.
Feel free to close this pr if you think it's inappropriate.

**Comment by yqlbu** [comment]
@czybjtu We appreciate the efforts you put in here. After some discussion, we decide to proceed with your proposed changes. However, we are thinking of including the `gofmt` as part of the pre-commit check. Would you mind holding this PR for a bit? cc @kunish @mzz2017

**Review by kunish — APPROVED** [review]
Looking good

File: Makefile [hunk 1]
```diff
@@ -43,6 +43,8 @@ dae: ebpf
 clean-ebpf: 
 	@rm -f control/bpf_bpf*.go && \
 		rm -f control/bpf_bpf*.o
+fmt:
+	go fmt ./...
```
Comment (yqlbu) [inline Makefile:47]: I've integrated this automation in `pre-commit`. Feel free to have a try afterward.

**Review by yqlbu — APPROVED** [review]

**Comment by yqlbu** [comment]
Looking good. Thanks for your contribution! @czybjtu

**Pull request merged**